src/main/java/com/maria/library/repository/mapper/RoleMapper.java

```diff
@@ -2,10 +2,12 @@
 
 import com.maria.library.model.Role;
 import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
 
 import java.sql.ResultSet;
 import java.sql.SQLException;
 
+@Component
```

Comment:
you don t need this class to be injected anywhere so you don' need this @Component annotation here